draft-dew-cfrg-signature-key-blinding.md

```diff
@@ -255,26 +255,30 @@ BlindKeySign(skS, skB, msg) works as follows:
 
 This section describes implementations of BlindPublicKey, UnblindPublicKey, and BlindKeySign as 
 functions implemented on top of an existing {{ECDSA}} implementation. In the descriptions below,
-let L be the order of the corresponding elliptic curve group used for ECDSA. For example, for 
-P-256, L = 2^256 - 2^224 + 2^192 + 2^96 - 1.
+let p be the order of the corresponding elliptic curve group used for ECDSA. For example, for 
+P-256, p = 2^256 - 2^224 + 2^192 + 2^96 - 1.
```

Comment:
This value is the prime modulus `p` of the curve, not its order `n`. (Cf. Section 4.2.1.3 https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-186-draft.pdf)

Reply:
Oops, existing bug. Thanks for flagging! (Also, do you have these memorized? 😆 )